mangaki/mangaki/admin.py

```diff
@@ -76,6 +191,10 @@ class WorkAdmin(admin.ModelAdmin):
         'controversy',
     )
 
+    def get_queryset(self, request):
+        qs = super().get_queryset(request)
+        return qs.filter(redirect__isnull=True)  # Exclude phantom works
```

Comment:
In [3]: WorkCluster.objects.get(id=10).works.all()
    Out[3]: <QuerySet [<Work: Patema Inverted>]>

    In [4]: WorkCluster.objects.filter(id=10).values_list('works')
    Out[4]: <QuerySet [(15427,), (15428,)]>

:'( C'est toujours filtré.